build/dockerfiles/dev.Dockerfile

```diff
@@ -6,81 +6,49 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-# https://quay.io/eclipse/che-machine-exec#^7\.
-FROM quay.io/eclipse/che-machine-exec:7.56.0 as machine-exec
+FROM quay.io/devfile/universal-developer-image:latest
 
-# https://registry.access.redhat.com/ubi8/ubi
-FROM registry.access.redhat.com/ubi8/ubi:8.6-990 AS ubi-micro-build
-RUN mkdir -p /mnt/rootfs
-RUN ARCH=$(uname -m) && yum install --installroot /mnt/rootfs libsecret curl make cmake gcc gcc-c++ python3 git git-core-doc openssh less wget \
-                 "https://rpmfind.net/linux/centos/8-stream/BaseOS/x86_64/os/Packages/libsecret-devel-0.18.6-1.el8.x86_64.rpm" \
-                 "https://rpmfind.net/linux/centos/8-stream/AppStream/x86_64/os/Packages/libxkbfile-1.1.0-1.el8.x86_64.rpm" \
-                 "https://rpmfind.net/linux/centos/8-stream/PowerTools/x86_64/os/Packages/libxkbfile-devel-1.1.0-1.el8.x86_64.rpm" \
-                 procps vi nano \
-                 "https://rpmfind.net/linux/centos/8-stream/BaseOS/${ARCH}/os/Packages/zsh-5.5.1-6.el8_1.2.x86_64.rpm" \
```

Comment:
Why do we need to remove zsh, kubectl from the Dev image?
It was added intentionally (by https://github.com/che-incubator/che-code/pull/21), for dev purposes

Reply:
okay, kubectl is pre-installed in UDI

Reply:
Ok, it's not a problem to get it back.
But I want to say you by a secret, `$HOME/.zshrc` is never called.
That was probably a reason why I removed zsh.

Reply:
It's called currently in quay.io/che-incubator/che-code-dev:insiders

Reply:
It's called currently in quay.io/che-incubator/che-code-dev:insiders